vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -385,17 +385,18 @@ - (void)listActivityInfoFromStart:(unsigned int)start
 
 - (void)activityInfoWithFilter:(nullable BATActivityInfoFilter *)filter completion:(void (^)(BATPublisherInfo * _Nullable info))completion
 {
-  const auto cppFilter = filter ? filter.cppObj : ledger::ActivityInfoFilter();
-  ledger->GetActivityInfo(cppFilter, ^(ledger::Result result, ledger::PublisherInfoPtr info) {
```

Comment:
We can remove the entire method here (and it's declaration in the header file if it exists)